### ISSUE TITLE
Recording of screen zoom in 

### ISSUE THREAD

**Issue by pmoneyyy** [joaomoreno/gifcap#89]
When i use the tool to record my screen when I stop recording the video is zoom into the top left corner and is not useable. attached is an example 
![Recording 2023-05-08 at 14 03 27](https://user-images.githubusercontent.com/132939920/236897544-f031d9da-828b-4c92-969f-25cd3f468e8f.gif)

**Comment by joe-lacasse** [comment]
I am seeing the same issue when recording a specific tab.

For what it's worth, sharing an entire screen worked fine for me as a workaround. (@pmoneyyy)

**Comment by Leventesz** [comment]
Yep, same thing here! Sharing the entire screen has the same issue for me.

**Comment by MWSDI** [comment]
This issue is still occurring - any fixes other than full screen?

**Comment by aaron-neal** [comment]
Also getting this issue. Full screen works fine.

**Comment by coiiii** [comment]
I have the same issue. I tried EDGE, but it doesn't work. Can you plz fix it?

**Comment by AlexandrevH82** [comment]
Have the same issue here in Edge only when capturing through sharing a single Edge TAB.
If you separate a tab into a its own window, it works (but you get all the window banner with it)

**Comment by evelynnmimijae** [comment]
I also experience a zoom issue with Chrome. I always have to screen record
the entire window instead of one tab.

**Comment by develop084** [comment]
I also got this issue, what I have done just refreshed the browser and start recording again it worked after that !

**Comment by qoreycube** [comment]
This possibly has something to do with meta viewport tags. I noticed the same thing happening to me, and it was completely frustrating... 

On a whim, I went into dev tools, found the head tag and removed the <meta name="viewport" content="width=device-width, initial-scale=1"> tag from the website I was trying to cap, and it suddenly started not zooming in.
